### PR TITLE
Fix some more flakiness

### PR DESCRIPTION
**Pull request by mariano54**
Decreases probability of failures

### CODE REVIEW

**Review by arvidn — APPROVED** [review]
did all the tests pass on the first pass with this patch? We don't really have a good metric for flakiness..

**Comment by mariano54** [comment]
> did all the tests pass on the first pass with this patch? We don't really have a good metric for flakiness..

The tests that I modified, did pass on the first try (and I ran it several times to fix other issues). 

However I saw another 4 or 5 other flaky tests that are more rare and I couldn't solve or didn't know how to solve easily. So we are a long way from full green. Some of these are:
- Farmer harvester port issue (for the tests that allocates it's port early)
- Windows file issues
- Routes do not exist for RPC server 
- File keyring OS issue

**Review by altendky — DISMISSED** [review]

File: tests/wallet/test_wallet.py [hunk 1]
```diff
@@ -193,19 +193,17 @@ async def test_wallet_coinbase_reorg(
...
-        "trusted",
-        [True, False],
-    )
+    @pytest.mark.parametrize("repeat", [i for i in range(100)])
     @pytest.mark.asyncio
     async def test_wallet_send_to_three_peers(
         self,
         three_sim_two_wallets: Tuple[List[FullNodeSimulator], List[Tuple[WalletNode, ChiaServer]]],
-        trusted: bool,
+        repeat: bool,
```
[the remark this answers is not in the document]
Comment (mariano54) [reply tests/wallet/test_wallet.py]: Oops yeah that was just to replicate the issue, will revert

File: tests/wallet/sync/test_wallet_sync.py [hunk 2]
```diff
@@ -160,9 +160,8 @@ async def test_basic_sync_wallet(self, bt, two_wallet_nodes, default_400_blocks,
 
         # Tests a reorg with the wallet
         num_blocks = 30
-        blocks_reorg = bt.get_consecutive_blocks(
-            num_blocks, block_list_input=default_400_blocks[:-5], current_time=True
-        )
+        blocks_reorg = bt.get_consecutive_blocks(num_blocks - 1, block_list_input=default_400_blocks[:-5])
+        blocks_reorg = bt.get_consecutive_blocks(1, blocks_reorg, guarantee_transaction_block=True, current_time=True)
```
[the remark this answers is not in the document]
Comment (mariano54) [reply tests/wallet/sync/test_wallet_sync.py:164]: No, the function returns all blocks starting at height 0,always.

File: tests/wallet/test_wallet.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariano54) [reply tests/wallet/test_wallet.py]: This comment seems to be on an outdated commit, so it's fine i think

**Pull request merged**